cabal-install/src/Distribution/Client/Config.hs

```diff
@@ -1784,7 +1784,7 @@ showConfigWithComments comment vals =
 
     -- skip fields based on field name.  currently only skips "remote-repo",
     -- because that is rendered as a section.  (see 'ppRemoteRepoSection'.)
```

Comment:
Seems like the comment should be updated as well.

Reply:
Addressed.